tests/integration/targets/azure_rm_datalakestore/aliases

```diff
@@ -0,0 +1,3 @@
+cloud/azure
+shippable/azure/group2
+destructive
```

Comment:
@next-davidduquehernandez The datalake resource only can create in some location, Please changed as follow! Thank you much!
```suggestion
destructive
disabled
```